docs/README-PT-PT.md

```
@@ -0,0 +1,556 @@
+- **Discord RPC**: RPC incorporado incluído!!!
+- **Múltiplos Backends**: Transmita a partir do seu servidor Jellyfin ou compatível com Subsonic
+  (Navidrome funciona muito bem), ligue o YouTube Music ou SoundCloud, ou simplesmente aponte-o
+  para uma pasta local. Misture e combine como quiser. Cada fonte é exposta através
+  de uma camada unificada `MediaSource`, e a interface adapta-se às
+  capacidades de cada fonte (pesquisa, transferências, rádio, descoberta, sincronização de favoritos, etc.) em vez de
+  codificar rigidamente o comportamento por serviço.
+- **YouTube Music**: Backend de streaming completo com uma página **Descobrir** no estilo Spotify (músicas recomendadas, playlists, álbuns, artistas e ambientes), perfis ricos de **artistas**
+  (banner, melhores músicas, álbuns, singles, artistas relacionados),
+  navegação por álbuns/playlists, e **rádio de mistura** ("iniciar rádio" a partir de qualquer faixa).
+  Inicie sessão com a sua conta para a sua biblioteca, Músicas Gostadas e playlists — ou
+  execute-o **anonimamente** (sem início de sessão) para navegar, pesquisar e reproduzir faixas públicas. Veja [Configuração do YouTube Music](#youtube-music-setup).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)